mix.exs

```diff
@@ -69,12 +70,29 @@ defmodule PropCheck.Mixfile do
   # Type `mix help deps` for more examples and options
   defp deps do
     [
-      {:dialyxir, "~> 0.5.1", only: [:dev, :test]},
-      {:coverex, "~> 1.4.15", only: :test},
-      {:ex_doc, "~>0.16", only: :dev},
-      {:earmark, ">= 0.2.1", only: :dev},
       {:proper, "~> 1.3"},
-      {:credo, "~> 1.0.0", only: [:dev, :test], runtime: false}
+      {:coverex, "~> 1.4", only: :test},
+      {:poison, "~> 3.0", only: :test},
+      {:credo, "~> 1.1", only: [:dev, :test], runtime: false},
+      {:dialyxir, "~> 1.0.0-rc.7", only: [:dev, :test], runtime: false},
```

Comment:
When is a 1.0 version to be expected? Any news on that? I would like to stick to a stable version.

Reply:
Thanks for the feedback, @alfert.

I don't know when they plan to have a `1.0` "stable" release.

But they've been on `1.0.0-rc.x` since 2018, and last version before that (`0.5.1` is from July 2017).

The tool got much better since then, works more nicely with Elixir, I guess they just don't make it `1.0.0` to try resolving some [other issues](https://github.com/jeremyjh/dialyxir/issues) before making it officially "stable".

I'd say rc it's much more stable than 0.5. `rc.7` is actually out since September, me and my team use it in dozens of projects without problems. Last (only) bug I've had with it was when rc.5 got out, and it got resolved really quickly https://github.com/jeremyjh/dialyxir/issues/310

Reply:
Passing the question along to them as it's indeed a very good point: https://github.com/jeremyjh/dialyxir/issues/379